src/CreatorToken.sol

```diff
@@ -30,14 +44,19 @@ contract CreatorToken is ERC721Royalty {
   IERC20 public payToken;
   IBondingCurve public immutable BONDING_CURVE;
 
+  /// @dev Constant to represent percentages in basis points.
   uint256 constant BIP = 10_000;
   uint256 public immutable CREATOR_FEE_BIPS;
   uint96 public immutable CREATOR_ROYALTY_BIPS;
   uint256 public immutable ADMIN_FEE_BIPS;
+  /// @dev Maximum allowed fee in basis points.
   uint256 private constant MAX_FEE = 2500; // 25% in bips
+  /// @dev Minimum time a user must hold a token before selling, in blocks.
   uint256 private constant MIN_HOLDING_TIME = 60;
+  /// @dev Mapping to track purchase time of tokens.
   mapping(uint256 tokenId => uint256 blockTimestamp) internal purchaseTime;
 
+  /// @dev Event emitted when a new token is bought.
   event Bought(
     address indexed payer,
```

Comment:
There's no natspec convention for this, but we should add a short comment on each parameter explaining it. For example:

```solidity
event Bought(
    address indexed payer, // The account that paid for the token
     address indexed receiver, // The account that received the newly minted token
    uint256 indexed tokenId, // etc...
    uint256 paymentAmount, // etc...
    uint256 creatorFee,
    uint256 adminFee
  );
```

Reply:
my formatter didn't like that, so used `@param`s instead https://github.com/showtime-xyz/creator-tokens/pull/46/commits/3c46721e1c9a0899f54630e2b73028cd5bef159c